glib/Gir.toml

```diff
@@ -190,59 +186,16 @@ status = "generate"
     #not in 64bit lib for Windows
     ignore = true
     [[object.function]]
-    name = "get_user_name"
-    #manual pathbuf is_windows_utf8
-    manual = true
-    [[object.function]]
     pattern = "[gs]etenv"
```

Comment:
Should I get rid of this line too?

Reply:
No why? It's `manual = true` below. But I believe you can replace that with `string_type = "os_string"` and then the autogenerated variant would be the same as the current manual one, making it obsolete.

Reply:
oh, I see. thanks